pallets/omnipool/src/lib.rs

```diff
@@ -232,6 +235,9 @@ pub mod pallet {
 
 		/// Oracle price provider. Provides price for given asset. Used in remove liquidity to support calculation of dynamic withdrawal fee.
 		type ExternalPriceOracle: ExternalPriceProvider<Self::AssetId, EmaPrice, Error = DispatchError>;
+
+		/// Operation id provider for unified events
+		type AmmUnifiedEventSupport: IncrementalIdProvider<IncrementalIdType> + ExecutionTypeStack<IncrementalIdType>;
```

Comment:
this does not make much sense.

you added direct dependency on pallet_amm_support - yet you require to provide something that implements these traits. and that something is set again to AmmSupportPallet in the runtime.